### ISSUE TITLE
drush rr no longer there in 0.11.0, causes errors when you download it

### ISSUE THREAD

**Issue by labboy0276** [kalabox/kalabox#860]
Hello!

Just like the title says, drush rr is no longer there, weird things happen when you try and run it / download it, see below:

```
john@kalamuna:~/kalabox/apps/mrpc$ kbox drush rr
info: COMPOSE =>  0=--project-name, 1=mrpc, 2=--file, 3=/home/john/kalabox/apps/mrpc/kalabox-compose.yml, 4=--file, 5=/home/john/kalabox/apps/mrpc/kalabox-cli.yml, 6=--file, 7=/home/john/kalabox/apps/mrpc/kalabox-cli.yml, 8=run, 9=--entrypoint, 10=drush8, 11=--rm, 12=terminus, 13=rr, 14=--strict=0, mode=attach
Starting mrpc_data_1
The drush command 'rr' could not be found.  Run `drush cache-clear drush` to clear the commandfile cache if you have installed new   [error]
extensions.
info: DOCKER-COMPOSE =>  Run exited with code: 1
Unhandled rejection VError: code1err:more:
    at ChildProcess.<anonymous> (/usr/local/bin/lib/util/shell.js.jx:247:16)
    at ChildProcess.emit (events.js:85:9)
    at maybeClose (child_process.js:773:12)
    at Process.ChildProcess._handle.onexit (child_process.js:839:1)
From previous event:
    at Object.exports.spawn (/usr/local/bin/lib/util/shell.js.jx:211:10)
    at Object.sh (/usr/local/bin/plugins/kalabox-engine-docker/provider/docker/lib/bin.js.jx:92:20)
    at /usr/local/bin/plugins/kalabox-engine-docker/provider/docker/compose.js.jx:60:18
From previous event:
    at shCompose (/usr/local/bin/plugins/kalabox-engine-docker/provider/docker/compose.js.jx:42:6)
    at Object.run (/usr/local/bin/plugins/kalabox-engine-docker/provider/docker/compose.js.jx:341:12)
    at /usr/local/bin/plugins/kalabox-engine-docker/index.js.jx:260:24
    at arrayMap (/usr/local/bin/node_modules/lodash/index.js.jx:1405:25)
    at Function.map (/usr/local/bin/node_modules/lodash/index.js.jx:6709:14)
    at Object.run (/usr/local/bin/plugins/kalabox-engine-docker/index.js.jx:177:26)
From previous event:
    at /usr/local/bin/lib/engine.js.jx:471:29
From previous event:
    at Object.exports.run (/usr/local/bin/lib/engine.js.jx:470:4)
    at Object.task.func (/usr/local/bin/plugins/kalabox-cli/lib/tasks.js.jx:100:34)
    at Object.exports.run (/usr/local/bin/lib/core/tasks.js.jx:884:15)
    at processTask (/usr/local/bin/bin/kbox.js.jx:168:16)
    at Object.<anonymous> (/usr/local/bin/bin/kbox.js.jx:231:12)
    at process._tickCallback (node.js:803:1)
From previous event:
    at main (/usr/local/bin/bin/kbox.js.jx:230:4)
    at Object.<anonymous> (/usr/local/bin/bin/kbox.js.jx:239:1)
    at Module._compile (module.js:595:24)
    at readX (module.js:1488:3)
    at Object.Module._extensions..jx (module.js:1921:8)
    at Module.load (module.js:416:30)
    at Function.Module._load (module.js:380:8)
    at Module.require (module.js:451:15)
    at require (module.js:469:13)
    at Object.<anonymous> (_jx_source:17:1)
    at Module._compile (module.js:595:24)
    at Module.load (module.js:425:6)
    at Function.Module._load (module.js:380:8)
    at Function.Module.runMain (module.js:813:8)
    at startup (node.js:374:8)
    at node.js:1461:1
john@kalamuna:~/kalabox/apps/mrpc$ kbox drush dl registry_rebuild
info: COMPOSE =>  0=--project-name, 1=mrpc, 2=--file, 3=/home/john/kalabox/apps/mrpc/kalabox-compose.yml, 4=--file, 5=/home/john/kalabox/apps/mrpc/kalabox-cli.yml, 6=--file, 7=/home/john/kalabox/apps/mrpc/kalabox-cli.yml, 8=run, 9=--entrypoint, 10=drush8, 11=--rm, 12=terminus, 13=dl, 14=registry_rebuild, 15=--strict=0, mode=attach
Starting mrpc_data_1
Project registry_rebuild (7.x-2.3) downloaded to /usr/share/drush/commands/registry_rebuild.                                         [success]
Project registry_rebuild contains 0 modules: .
info: DOCKER-COMPOSE =>  Run exited with code: 0
john@kalamuna:~/kalabox/apps/mrpc$ kbox drush rr
info: COMPOSE =>  0=--project-name, 1=mrpc, 2=--file, 3=/home/john/kalabox/apps/mrpc/kalabox-compose.yml, 4=--file, 5=/home/john/kalabox/apps/mrpc/kalabox-cli.yml, 6=--file, 7=/home/john/kalabox/apps/mrpc/kalabox-cli.yml, 8=run, 9=--entrypoint, 10=drush8, 11=--rm, 12=terminus, 13=rr, 14=--strict=0, mode=attach
Starting mrpc_data_1
The drush command 'rr' could not be found.  Run `drush cache-clear drush` to clear the commandfile cache if you have installed new   [error]
extensions.
info: DOCKER-COMPOSE =>  Run exited with code: 1
Unhandled rejection VError: code1err:more:
    at ChildProcess.<anonymous> (/usr/local/bin/lib/util/shell.js.jx:247:16)
    at ChildProcess.emit (events.js:85:9)
    at maybeClose (child_process.js:773:12)
    at Process.ChildProcess._handle.onexit (child_process.js:839:1)
From previous event:
    at Object.exports.spawn (/usr/local/bin/lib/util/shell.js.jx:211:10)
    at Object.sh (/usr/local/bin/plugins/kalabox-engine-docker/provider/docker/lib/bin.js.jx:92:20)
    at /usr/local/bin/plugins/kalabox-engine-docker/provider/docker/compose.js.jx:60:18
From previous event:
    at shCompose (/usr/local/bin/plugins/kalabox-engine-docker/provider/docker/compose.js.jx:42:6)
    at Object.run (/usr/local/bin/plugins/kalabox-engine-docker/provider/docker/compose.js.jx:341:12)
    at /usr/local/bin/plugins/kalabox-engine-docker/index.js.jx:260:24
    at arrayMap (/usr/local/bin/node_modules/lodash/index.js.jx:1405:25)
    at Function.map (/usr/local/bin/node_modules/lodash/index.js.jx:6709:14)
    at Object.run (/usr/local/bin/plugins/kalabox-engine-docker/index.js.jx:177:26)
From previous event:
    at /usr/local/bin/lib/engine.js.jx:471:29
From previous event:
    at Object.exports.run (/usr/local/bin/lib/engine.js.jx:470:4)
    at Object.task.func (/usr/local/bin/plugins/kalabox-cli/lib/tasks.js.jx:100:34)
    at Object.exports.run (/usr/local/bin/lib/core/tasks.js.jx:884:15)
    at processTask (/usr/local/bin/bin/kbox.js.jx:168:16)
    at Object.<anonymous> (/usr/local/bin/bin/kbox.js.jx:231:12)
    at process._tickCallback (node.js:803:1)
From previous event:
    at main (/usr/local/bin/bin/kbox.js.jx:230:4)
    at Object.<anonymous> (/usr/local/bin/bin/kbox.js.jx:239:1)
    at Module._compile (module.js:595:24)
    at readX (module.js:1488:3)
    at Object.Module._extensions..jx (module.js:1921:8)
    at Module.load (module.js:416:30)
    at Function.Module._load (module.js:380:8)
    at Module.require (module.js:451:15)
    at require (module.js:469:13)
    at Object.<anonymous> (_jx_source:17:1)
    at Module._compile (module.js:595:24)
    at Module.load (module.js:425:6)
    at Function.Module._load (module.js:380:8)
    at Function.Module.runMain (module.js:813:8)
    at startup (node.js:374:8)
    at node.js:1461:1
john@kalamuna:~/kalabox/apps/mrpc$ 
```

**Comment by pirog** [comment]
Does drush rr exist for drush 8?

On Saturday, January 23, 2016, John Ouellet <notifications@github.com>
wrote:

> Hello!
>
> Just like the title says, drush rr is no longer there, weird things happen
> when you try and run it / download it, see below:
>
> john@kalamuna:~/kalabox/apps/mrpc$ kbox drush rr
> info: COMPOSE =>  0=--project-name, 1=mrpc, 2=--file, 3=/home/john/kalabox/apps/mrpc/kalabox-compose.yml, 4=--file, 5=/home/john/kalabox/apps/mrpc/kalabox-cli.yml, 6=--file, 7=/home/john/kalabox/apps/mrpc/kalabox-cli.yml, 8=run, 9=--entrypoint, 10=drush8, 11=--rm, 12=terminus, 13=rr, 14=--strict=0, mode=attach
> Starting mrpc_data_1
> The drush command 'rr' could not be found.  Run `drush cache-clear drush` to clear the commandfile cache if you have installed new   [error]
> extensions.
> info: DOCKER-COMPOSE =>  Run exited with code: 1
> Unhandled rejection VError: code1err:more:
>     at ChildProcess.<anonymous> (/usr/local/bin/lib/util/shell.js.jx:247:16)
>     at ChildProcess.emit (events.js:85:9)
>     at maybeClose (child_process.js:773:12)
>     at Process.ChildProcess._handle.onexit (child_process.js:839:1)
> From previous event:
>     at Object.exports.spawn (/usr/local/bin/lib/util/shell.js.jx:211:10)
>     at Object.sh (/usr/local/bin/plugins/kalabox-engine-docker/provider/docker/lib/bin.js.jx:92:20)
>     at /usr/local/bin/plugins/kalabox-engine-docker/provider/docker/compose.js.jx:60:18
> From previous event:
>     at shCompose (/usr/local/bin/plugins/kalabox-engine-docker/provider/docker/compose.js.jx:42:6)
>     at Object.run (/usr/local/bin/plugins/kalabox-engine-docker/provider/docker/compose.js.jx:341:12)
>     at /usr/local/bin/plugins/kalabox-engine-docker/index.js.jx:260:24
>     at arrayMap (/usr/local/bin/node_modules/lodash/index.js.jx:1405:25)
>     at Function.map (/usr/local/bin/node_modules/lodash/index.js.jx:6709:14)
>     at Object.run (/usr/local/bin/plugins/kalabox-engine-docker/index.js.jx:177:26)
> From previous event:
>     at /usr/local/bin/lib/engine.js.jx:471:29
> From previous event:
>     at Object.exports.run (/usr/local/bin/lib/engine.js.jx:470:4)
>     at Object.task.func (/usr/local/bin/plugins/kalabox-cli/lib/tasks.js.jx:100:34)
>     at Object.exports.run (/usr/local/bin/lib/core/tasks.js.jx:884:15)
>     at processTask (/usr/local/bin/bin/kbox.js.jx:168:16)
>     at Object.<anonymous> (/usr/local/bin/bin/kbox.js.jx:231:12)
>     at process._tickCallback (node.js:803:1)
> From previous event:
>     at main (/usr/local/bin/bin/kbox.js.jx:230:4)
>     at Object.<anonymous> (/usr/local/bin/bin/kbox.js.jx:239:1)
>     at Module._compile (module.js:595:24)
>     at readX (module.js:1488:3)
>     at Object.Module._extensions..jx (module.js:1921:8)
>     at Module.load (module.js:416:30)
>     at Function.Module._load (module.js:380:8)
>     at Module.require (module.js:451:15)
>     at require (module.js:469:13)
>     at Object.<anonymous> (_jx_source:17:1)
>     at Module._compile (module.js:595:24)
>     at Module.load (module.js:425:6)
>     at Function.Module._load (module.js:380:8)
>     at Function.Module.runMain (module.js:813:8)
>     at startup (node.js:374:8)
>     at node.js:1461:1
> john@kalamuna:~/kalabox/apps/mrpc$ kbox drush dl registry_rebuild
> info: COMPOSE =>  0=--project-name, 1=mrpc, 2=--file, 3=/home/john/kalabox/apps/mrpc/kalabox-compose.yml, 4=--file, 5=/home/john/kalabox/apps/mrpc/kalabox-cli.yml, 6=--file, 7=/home/john/kalabox/apps/mrpc/kalabox-cli.yml, 8=run, 9=--entrypoint, 10=drush8, 11=--rm, 12=terminus, 13=dl, 14=registry_rebuild, 15=--strict=0, mode=attach
> Starting mrpc_data_1
> Project registry_rebuild (7.x-2.3) downloaded to /usr/share/drush/commands/registry_rebuild.                                         [success]
> Project registry_rebuild contains 0 modules: .
> info: DOCKER-COMPOSE =>  Run exited with code: 0
> john@kalamuna:~/kalabox/apps/mrpc$ kbox drush rr
> info: COMPOSE =>  0=--project-name, 1=mrpc, 2=--file, 3=/home/john/kalabox/apps/mrpc/kalabox-compose.yml, 4=--file, 5=/home/john/kalabox/apps/mrpc/kalabox-cli.yml, 6=--file, 7=/home/john/kalabox/apps/mrpc/kalabox-cli.yml, 8=run, 9=--entrypoint, 10=drush8, 11=--rm, 12=terminus, 13=rr, 14=--strict=0, mode=attach
> Starting mrpc_data_1
> The drush command 'rr' could not be found.  Run `drush cache-clear drush` to clear the commandfile cache if you have installed new   [error]
> extensions.
> info: DOCKER-COMPOSE =>  Run exited with code: 1
> Unhandled rejection VError: code1err:more:
>     at ChildProcess.<anonymous> (/usr/local/bin/lib/util/shell.js.jx:247:16)
>     at ChildProcess.emit (events.js:85:9)
>     at maybeClose (child_process.js:773:12)
>     at Process.ChildProcess._handle.onexit (child_process.js:839:1)
> From previous event:
>     at Object.exports.spawn (/usr/local/bin/lib/util/shell.js.jx:211:10)
>     at Object.sh (/usr/local/bin/plugins/kalabox-engine-docker/provider/docker/lib/bin.js.jx:92:20)
>     at /usr/local/bin/plugins/kalabox-engine-docker/provider/docker/compose.js.jx:60:18
> From previous event:
>     at shCompose (/usr/local/bin/plugins/kalabox-engine-docker/provider/docker/compose.js.jx:42:6)
>     at Object.run (/usr/local/bin/plugins/kalabox-engine-docker/provider/docker/compose.js.jx:341:12)
>     at /usr/local/bin/plugins/kalabox-engine-docker/index.js.jx:260:24
>     at arrayMap (/usr/local/bin/node_modules/lodash/index.js.jx:1405:25)
>     at Function.map (/usr/local/bin/node_modules/lodash/index.js.jx:6709:14)
>     at Object.run (/usr/local/bin/plugins/kalabox-engine-docker/index.js.jx:177:26)
> From previous event:
>     at /usr/local/bin/lib/engine.js.jx:471:29
> From previous event:
>     at Object.exports.run (/usr/local/bin/lib/engine.js.jx:470:4)
>     at Object.task.func (/usr/local/bin/plugins/kalabox-cli/lib/tasks.js.jx:100:34)
>     at Object.exports.run (/usr/local/bin/lib/core/tasks.js.jx:884:15)
>     at processTask (/usr/local/bin/bin/kbox.js.jx:168:16)
>     at Object.<anonymous> (/usr/local/bin/bin/kbox.js.jx:231:12)
>     at process._tickCallback (node.js:803:1)
> From previous event:
>     at main (/usr/local/bin/bin/kbox.js.jx:230:4)
>     at Object.<anonymous> (/usr/local/bin/bin/kbox.js.jx:239:1)
>     at Module._compile (module.js:595:24)
>     at readX (module.js:1488:3)
>     at Object.Module._extensions..jx (module.js:1921:8)
>     at Module.load (module.js:416:30)
>     at Function.Module._load (module.js:380:8)
>     at Module.require (module.js:451:15)
>     at require (module.js:469:13)
>     at Object.<anonymous> (_jx_source:17:1)
>     at Module._compile (module.js:595:24)
>     at Module.load (module.js:425:6)
>     at Function.Module._load (module.js:380:8)
>     at Function.Module.runMain (module.js:813:8)
>     at startup (node.js:374:8)
>     at node.js:1461:1
> john@kalamuna:~/kalabox/apps/mrpc$
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kalabox/kalabox/issues/860>.
>


-- 
Mike Pirog
Co-Founder, Kalabox Inc

**Comment by pirog** [comment]
@labboy0276 for now i would drop the rr command files in your drush config directory inside your app and see if that works. Would also take a PR to add this back in.

**Comment by labboy0276** [comment]
Negative on D8, doesnlt really apply per say to the way it is setup meow.

I assigned this to me, so I can remember to do a PR.

**Comment by pirog** [comment]
drush 8 not drupal8

**Comment by pirog** [comment]
@labboy0276 def try to drop the rr commandfiles into appRoot/config/drush and see if it will load it up. If this works for drush8 still lets add it back in. going to assign this to you since it seems pretty easy.

**Comment by labboy0276** [comment]
@pirog yes this solution worked in the interim

**Comment by pirog** [comment]
k, post a PR to roll RR back in if you think that is something we should do.

**Comment by labboy0276** [comment]
@pirog All set:

https://github.com/kalabox/kalabox-app-pantheon/pull/80

**Comment by pirog** [comment]
sweet. i merged this in for v0.11.1
